torchtitan/components/checkpoint.py

```diff
@@ -112,14 +112,19 @@ def purge_thread(purge_queue: queue.Queue):
             if isinstance(path, Terminate):
                 return
             assert isinstance(path, str)
-            logger.info("Checkpointer is deleting %s.", path)
+
+            if not 'ft-replica' in path:
```

Comment:
Can "ft-replica" be a variable instead and use that across the checks? Also it looks like there is a mispelling for the folder name since its "replicat" currently lol, can you update that?

https://github.com/pytorch/torchtitan/blob/92bea07501005b644dff3dd166f573f4991eaaa7/torchtitan/components/checkpoint.py#L628